docs/source/examples/sid_visualization/README.md

```diff
@@ -0,0 +1 @@
+../../../../examples/sid_visualization/README.md
```

Comment:
`Build the dev container` should be part of `SID Visualiation Example`.
<img width="300" alt="Screen Shot 2023-01-27 at 6 44 41 PM" src="https://user-images.githubusercontent.com/48110516/215226303-b39a53b2-c99c-4366-946a-9b3f882dffd5.png">